corehq/toggles.py

```diff
@@ -784,6 +795,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_PRODUCT_PATH,
     [NAMESPACE_DOMAIN],
     0.1,
+    always_disabled=['hsph-betterbirth'],
```

Comment:
:+1:

On Tue, Sep 6, 2016, 4:10 PM Ben Rudolph <notifications@github.com> wrote:

> In corehq/toggles.py
> <https://github.com/dimagi/commcare-hq/pull/13111#discussion_r77708849>:
>
> > @@ -784,6 +795,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
> >      TAG_PRODUCT_PATH,
> >      [NAMESPACE_DOMAIN],
> >      0.1,
> > +    always_disabled=['hsph-betterbirth'],
>
> it is indeed just the domain, i checked locally as well because i also get
> confused
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/13111/files/2c7cc831db130d17c8b998108adae2a1cc5075d8..b010f7777a56e6d10701a0d911b152badbf9b10c#r77708849>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIX_JeKviX62QXfKwA9GfrdJuyz9R2Cks5qnciqgaJpZM4J1dv->
> .
>